community-stabilization-fund-api/src/modules/checklists/components/DymoBagLabels/DymoBagList.tsx

```diff
@@ -0,0 +1,19 @@
+import DymoBagLabel from './DymoBaglabel';
+
```

Comment:
better to have one baglist component and it renders the diff bag label based on bagLabelType with if statements